go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/k8snetworkplumbingwg/sriov-cni
 
-go 1.17
+go 1.19
```

Comment:
this directive means the minimum go version to build this module is 1.19[1]

related to comment in buildtest.yaml

[1] https://go.dev/doc/modules/gomod-ref